testing/test_deploy.py

```diff
@@ -270,6 +299,42 @@ def main():  # pylint: disable=too-many-locals
 
   parser_teardown.set_defaults(func=teardown)
 
+  parser.add_argument(
+    "--deploy_core",
+    default=True,
+    type=bool,
+    help=("If True, deploy the kubeflow-core component."))
+
+  parser.add_argument(
```

Comment:
Why is are you adding an argument to deploy_tf_job? Where does this get used?

Reply:
removing